nmdc_schema/migrators/migrator_from_X_to_PR21.py

```diff
@@ -0,0 +1,45 @@
+from nmdc_schema.migrators.migrator_base import MigratorBase
+from nmdc_schema.migrators.helpers import load_yaml_asset
+
+class Migrator_from_X_to_PR21(MigratorBase):
+    """Migrates data from schema X to PR21"""
+
+    def __init__(self, *args, **kwargs) -> None:
+        """Invokes parent constructor and populates collection-to-transformations map."""
+
+        super().__init__(*args, **kwargs)
+
+        # Populate the "collection-to-transformers" map for this specific migration.
+        self.agenda = dict(
+            study_set=[self.move_relevant_protocols_to_protocol_link],
+        )
+
+    def move_relevant_protocols_to_protocol_link(self, study: dict):
+        r"""
+        Transforms `websites` values that are dois into curies and moves them into the `associated_dois slot
```

Comment:
I think the `associated_dois` is missing the closing backtick.

Reply:
Oops, this should not be here. I will fix the description. This is for an old migrator